docs/ECLStandardLibraryReference/SLR-Mods/MonitorLogicalFileName.xml

```diff
@@ -98,6 +97,10 @@
   <emphasis>event</emphasis> with the name of the triggering object as the
   event subtype (see the EVENT function).</para>
 
+  <para>This function does not support wildcard characters. To monitor for
+  files using wildcards, use the <link
```

Comment:
Also, this makes it sound like MonitorFile can be used for logical files, whereas MonitorFile is just for monitoring physical files/directories.